docs/sources/examples/mongodb/Dockerfile

```diff
@@ -7,8 +7,8 @@ MAINTAINER Docker
 
 # Installation:
 # Import MongoDB public GPG key AND create a MongoDB list file
-RUN apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv 7F0CEB10
-RUN echo 'deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen' | tee /etc/apt/sources.list.d/10gen.list
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
+RUN echo "deb http://repo.mongodb.org/apt/ubuntu "$(lsb_release -sc)"/mongodb-org/3.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.0.list
```

Comment:
I don't think sudo is required here, given that this is is executed as root inside the container/image

Have you tested this? Because looking at this, it looks like there's too many/nested quotes (`"`).

Generally, I'd prefer to use a static string here, instead of `$(lsb_release ....`; we *know* what we're running on in the Dockerfile, and that won't change.